### DISCUSSION TITLE
Possible to launch an app and search content

### DISCUSSION THREAD

**Discussion by dmsviento** [supersaiyanmode/PyWebOSTV#92] [Q&A]
Hi,

Firstly, thank you for the work, this has been a tremendous help for myself. Is it, or would it be possible to launch an app (for example Prime or Youtube) and rather than open specific content search for content? For example:

launch_info = app.launch(yt, search_content=<search string>)

Many thanks

**Comment by supersaiyanmode** [comment]
While it should be straightforward to add such a functionality (by simply adding a line here: https://github.com/supersaiyanmode/PyWebOSTV/blob/master/pywebostv/controls.py#L282), I don't think this library supports it at this time.

These parameters are app-specific, however for youtube specifically: can you try passing: `contentId=search_query=video`? If this works out, we will explore adding a new API that lets generically add parameters while launching the app.

**Reply by dmsviento** [reply]
Apologies for a very long delay in replying.

Unfortunately the search_query does not work
